media-sound/teamspeak-client-bin/teamspeak-client-bin-3.0.16.ebuild

```diff
@@ -20,6 +20,7 @@ IUSE="alsa pulseaudio"
 REQUIRED_USE="|| ( alsa pulseaudio )"
 
 RDEPEND="
+	dev-libs/quazip[qt5]
```

Comment:
Since this is a binary package, it would be nice to depend on a specific subslot (since no other subslot will work). But I don't see a subslot declared in quazip, so I'm going to ask @jlec.

@jlec, can we assume that subslot `:0/0` will have libquazip.so.1, or do you plan to change that to `:0/1`?